src/Components/forms/index.js

```diff
@@ -101,6 +101,11 @@ class Forms extends Component {
       this.setState({ hearAboutCYFFromEmployer: value === 'Employer' })
     }
   }
+  employerOnChange = employer => {
+    const { errors } = this.state
+    errors[`${'Employer'}`] = false
+    this.setState({ employer, errors, submitted: false, formInComplete: false })
```

Comment:
You don't need to interpolate a string into a template string, just use the string:
```suggestion
    errors['Employer'] = false
    this.setState({ employer, errors, submitted: false, formInComplete: false })
```
and then hopefully it's more obvious that this is equivalent to:
```suggestion
    errors.Employer = false
    this.setState({ employer, errors, submitted: false, formInComplete: false })
```
In general you shouldn't mutate state held by React, but it looks like the other handlers do this too and enough is changing that it triggers a re-render, so maybe that's OK.

Reply:
applied the required changes